### ISSUE TITLE
Build a RaspberryPi with webcam for streaming GS

### ISSUE THREAD

**Issue by comzeradd** [satnogs/satnogs-org#56]

**Comment by acinonyx** [comment]
Hardware arrived back to hackerspace. Should be functional. Someone needs to test it.

**Comment by ppapadeas** [comment]
Marios is working on it

**Comment by ppapadeas** [comment]
Pinged Marios again

**Comment by papachristoumarios** [comment]
Yes, but I am busy with school + fixing some minor bugs (+ documentating) and many more (related to other projects). I'll inform you upon updates. I have also uploaded some 3D designs for the sensor enclosure under the `dev` branch at http://github.com/papachristoumarios/satnogs-meteo (I should transfer ownership to `satnogs` organization some time, shouldn't I?). You can check the code yourself / yourselves, as well, at the `dev` branch (I have added direct support for I2C / IO between the sensors and RPi).
